### ISSUE TITLE
Gr00t DiT Input

### ISSUE THREAD

**Issue by andrearosasco** [huggingface/lerobot#2571]
Hello, I have a question of the input to the diffusion transformer in the implementation of `Gr00tN1.5`.
I noticed that the input is constructed as `sa_embs = torch.cat((state_features, future_tokens, action_features), dim=1)` where `future_token` is initialized as `self.future_tokens = nn.Embedding(config.num_target_vision_tokens, self.input_embedding_dim)`. Is this a leftover from [FLARE](https://research.nvidia.com/labs/gear/flare/)?

**Comment by ulost2527** [comment]
Hello @andrearosasco , I was also curious about that var "future_tokens". I thought it was from FLARE project. But as you know FLARE is not applied to groot this ver. Do you know anything else?

**Comment by andrearosasco** [comment]
Hey @ulost2527, yeah I think it is most definitely FLARE. Actually I think it is used in Gr00t N1.5 pre-training, they just didn't release the code to use it for fine-tuning too. 

I only would have expected the conditioning of the Gr00t DiT to be done through 32 tokens obtained by applying Q-Former to eagle output tokens but apparently it is not the case. So I guess it is a bit different from FLARE where the target VLM embedding network has the same architecture of the one used by the policy.
